src/components/AccessibilityGuide/ProgressiveEnhancement.mdx

```diff
@@ -0,0 +1,109 @@
+import { Meta, Source } from "@storybook/blocks";
+
+import Link from "../Link/Link";
+
+<Meta title="Accessibility Guide/Progressive Enhancement" />
+
+# Progressive Enhancement
+
+## Table of Contents
+
+- {<Link href="#general-information" target="_self">General Information</Link>}
+- {<Link href="#nypl-and-ds-problem" target="_self">NYPL and DS Problem</Link>}
+- {<Link href="#html-button-issue" target="_self">HTML Button Issue</Link>}
+- {<Link href="#component-audit" target="_self">Component Audit</Link>}
+- {<Link href="#general-recommendations" target="_self">General Recommendations</Link>}
+- {<Link href="#resources" target="_self">Resources</Link>}
+
+## General Information
+
+Progressive enhancement is a frontend development strategy and mindset that
+emphasizes building a "basic", functional version of a website or component first,
+and then adding more advanced features and enhancements. This approach ensures
+that the core functionality of a component or site is accessible to all users,
+regardless of their device or browser capabilities. Once the necessary functionality
+is in place, we can provide additional enhancements for users with modern browsers
+and devices.
+
+This approach is particularly important for accessibility and users with
+disabilities, who may be using assistive technologies or older devices that do
+not support advanced features. This inclusive design philosophy also focuses on
+developers knowing and implement web fundamentals, such as HTML and CSS, before
+heavily relying on JavaScript.
+
+## NYPL and DS Problem
+
+When Javascript is turned off in a browser, certain Design System (DS) components
+and general user interfaces do not work. There are various reasons why app-level
+features may not work including how the application is architected, how data is
+fetched, and how data is rendered. Many issues, however, point back to DS
+components.
```

Comment:
Nit, slight modification for clarity:

"When Javascript is turned off in the browser, certain NYPL user interfaces and features do not work. There are various reasons why this is, including how the application is architected, how data is fetched, and how data is rendered. Many issues, however, point back to DS components."